src/utils.js

```diff
@@ -31,6 +33,24 @@ function defaultPortalUrl() {
   return defaultSkynetPortalUrl;
 }
 
+/**
+ * Get the file mime type. Try to guess the file type based on the extension.
+ *
+ * @param filename - The filename.
+ * @returns - The mime type.
+ */
+function getFileMimeType(filename) {
+  let { ext } = path.parse(filename);
+  ext = trimPrefix(ext, ".");
+  if (ext !== "") {
+    const mimeType = mime.getType(ext);
+    if (mimeType) {
+      return mimeType;
+    }
+  }
+  return "";
```

Comment:
Should we default to an empty string or `application/octet-stream`?

Reply:
There was a discussion about that here https://stackoverflow.com/questions/1176022/unknown-file-type-mime

Most upvoted answer is "You can use application/octet-stream for unknown types." but the other one says:

> RFC resources:
We should use RFC-7231 (HTTP/1.1 Semantics and Content) as reference instead of RFC-2046 (Media Types) because question was clearly about HTTP Content-Type.
>
> Also RFC-2046 does not clearly define unknown types but RFC-7231 does.
>
> Short answer:
Do not send MIME type for unknown data.
To be more clear: Do not use Content-Type header at all.

I would side with not setting the mime type at all for unknown type.

Reply:
Works for me. :)

Reply:
I think what Karol means is that if `getFileMimeType(filename)` is an empty string, we should simply not pass it along with the metadata under the `filetype` property. Providing that's an option of course...

so:
```
const tusOpts = {
      ....
      metadata: {
        filename,
        filetype: getFileMimeType(filename),
      },
      ...
}
```
simply becomes:
```
const filetype = getFileMimeType(filename)
const metadata = filetype ? { filename, filetype } : { filename }
const tusOpts = {
      ....
      metadata,
      ...
}
```